pkgs/by-name/wi/wifi-qr/package.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, fetchFromGitHub
+, installShellFiles
+, makeWrapper
+, gnome
+, ncurses
+, networkmanager
+, patsh
+, qrencode
+, stdenvNoCC
+, xdg-utils
+, zbar
+}:
+stdenvNoCC.mkDerivation {
+  pname = "wifi-qr";
+  version = "unstable-2023-09-30";
```

Comment:
Hope you don't mind me commenting on the version again.

While the release of `0.3` was made at 30th of September, I see you're pointing to a commit made the same day 9 commits ahead. To follow the (seemingly new) convention, it should probably be versioned at `0.3-unstable-2023-09-30`.